File: src/main/java/com/gimranov/zandy/app/model/Library.java

package com.gimranov.zandy.app.model;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Library implements Serializable
{

    @SerializedName("type")
    @Expose
    public String type;
    @SerializedName("id")
    @Expose
    public int id;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("links")
    @Expose
    public Links links;
    private final static long serialVersionUID = 3455999056633603093L;

}

File: src/main/java/com/gimranov/zandy/app/model/Library.kt
package com.gimranov.zandy.app.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Library(

        @SerializedName("type")
        @Expose
        var type: String? = null,
        @SerializedName("id")
        @Expose
        var id: Int = 0,
        @SerializedName("name")
        @Expose
        var name: String? = null,
        @SerializedName("links")
        @Expose
        var links: Links? = null
)